tests/debugger/bug01753-PageIndex.inc

```diff
@@ -1,12 +1,12 @@
-<?php
-class PageIndex extends SkeletonPage
-{
-
-
-
-
-	public function getData()
-	{
-		$var = 1;
-	}
-}
\ No newline at end of file
+<?php
+class PageIndex extends SkeletonPage
+{
+
+
+
+
+	public function getData()
+	{
+		$var = 1;
+	}
+}
```

Comment:
This seems an unrelated change, and hence don't add it to the commits in this PR.

Reply:
I opened https://github.com/xdebug/xdebug/pull/605 for it, I will then rebase this PR